node-operators/run-aut/index.md

```diff
@@ -53,40 +104,99 @@ Autonity will download the blockchain in "snap" syncmode by default.  Once fully
+	```
+
+::: {.callout-important title="Important" collapse="false"}
+Remember to backup your `autonitykeys` file! Copy it to a safe location!
+:::
+
+::: {.callout-important title="Important" collapse="false"}
+
+Copy the output - you will need the output if you intend to register your validator as a node.
+
+It is *not* possible to inspect the keyfile to retrieve the `Consensus public key` which you MUST provide as part of the validator registration information.
+
+:::
+
+
+4. Start the node. Set the Docker configuration and the arguments for connecting Autonity to a network.
 
    ```bash
    docker run \
        -t -i \
        --volume $(pwd)/autonity-chaindata:/autonity-chaindata \
+       --volume $(pwd)/<DIR_PATH>:/<DIR_PATH>
        --publish 8545:8545 \
        --publish 8546:8546 \
        --publish 30303:30303 \
        --publish 30303:30303/udp \
+       --publish 20203:20203 \
        --publish 6060:6060 \
        --name autonity \
        --rm \
        ghcr.io/autonity/autonity:latest \
            --datadir ./autonity-chaindata  \
+           --autonitykeys ./<DIR_PATH>/<AUTONITYKEYS_FILE_NAME> \
            --piccadilly \
            --http  \
            --http.addr 0.0.0.0 \
            --http.api aut,eth,net,txpool,web3,admin  \
-           --http.vhosts \* \
+           --http.vhosts "*" \
            --ws  \
            --ws.addr 0.0.0.0 \
            --ws.api aut,eth,net,txpool,web3,admin  \
-           --nat extip:<IP_ADDRESS>
+           --nat extip:<IP_ADDRESS> \
+           --consensus.nat <CONSENSUS_NAT> \
+           --consensus.port <CONSENSUS_PORT_NUMBER> \
+        ;
    ```
 
    where:
+   
+   - `<DIR_PATH>` is the path to the directory where you will store the `autonitykeys` private key file of the node.
+   - `<AUTONITYKEYS_FILE_NAME>` is the path to the `autonitykeys` file generated in Step 2.
    - `<IP_ADDRESS>` is the node's host IP Address, which can be determined with `curl ifconfig.me`.
-   - `--piccadilly` specifies that the node will use the Piccadilly tesnet.  For other tesnets, use the appropriate flag (for example, `--bakerloo`).
+   - (Optional) `<CONSENSUS_NAT>` specify the NAT port mapping for the consensus channel (one of "any", "none", "upnp", "pmp", "extip:<IP>") if the default value "any" is not to be used.
+   - (Optional) `<CONSENSUS_PORT_NUMBER>` specify the network listening port for the consensus channel if the default port "20203" is not to be used.
+   - `--piccadilly` specifies that the node will connect to the Piccadilly testnet.  For other testnets, use the appropriate flag (for example, `--bakerloo`).
+
+::: {.callout-warning title="Warning!" collapse="false"}
+You *must* specify the `--autonitykeys` command option.
+
+If it is *not* specified, then on starting AGC will automatically generate a standard Ethereum `nodekey` file  by default within the `autonity` subfolder of the `--datadir` specified when running the node. 
```

Comment:
Same as my last comment.

Reply:
Done, per response to your last comment.